tests/core/test_geometric_objects.py

```diff
@@ -35,11 +35,57 @@ def test_sphere():
     assert np.any(surf.points)
     assert np.any(surf.faces)
 
+    # test direction points from south pole to north pole
```

Comment:
Let's define a new test function to make it easier to understand what you are doing by the name of the test.

Reply:
```suggestion
def test_sphere_direction_points():
    # from south pole to north pole
```